decidim-admin/app/views/decidim/admin/organization_appearance/form/_colors.html.erb

```diff
@@ -1,6 +1,16 @@
 <div class="card">
   <div class="card-divider">
-    <h2 class="card-title"><%= t ".colors_title" %></h2>
+    <h2 class="card-title"><%= t ".colors_title" %> </h2>
+  </div>
+  <div class="callout warning" data-closable>
+    <svg role="img" class="icon--warning icon">
```

Comment:
We've actually a helper for these cases. You can see it in action in [/admin/consultations](https://github.com/decidim/decidim/blob/91f47299196b96e80c64c4e3055d74f19d9edc16/decidim-consultations/app/views/layouts/decidim/admin/consultations.html.erb#L9).